server/src/main/java/io/druid/segment/realtime/plumber/RealtimePlumber.java

```diff
@@ -363,7 +363,9 @@ private Sink getSink(long timestamp)
                                           }
                                         }
                                     )
-                                )
+                                ),
+                                "query/wait/time",
+                                ImmutableMap.of("segment", theSink.getSegment().getIdentifier())
```

Comment:
query/realtimePartialSegment/time?

I couldnt think of a good name which is why I probably left it as 'partial'